### PR TITLE
Adding meeting minutes for 2017-12-14

### PR DESCRIPTION
**Pull request by bnb**

### CODE REVIEW

File: meetings/2017-12-14.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+* Sarah Conway
+* Manil Chowdhury @chowdhurian
+* Tierney Cyren @bnb
+
+## Notes
+
+Extracted from cc-agenda labelled issues and pull requests from the nodejs org prior to the meeting.
+Extracted from **cc-agenda** labelled issues and pull requests from the **nodejs org** prior to the meeting.
+
+Mentorship Team under the CommComm #172
```
Comment (pup) [inline meetings/2017-12-14.md]: label these as bullet points, otherwise markdown will concat them into a single line

**Comment by bnb** [comment]
@pup 100% agreed. Asking someone to PR them in during the meeting may be a good idea - will try that today :thumb

File: meetings/2017-12-14.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+* Pairing done by commcomm participants.
+* Factor in language preferences
+* Add info to repo README as well as have slack to communicate
+* Restrict mentorship to a given timeline, suggested 6 months is a good timeline which
+  gives around 10-12 meetings.
+
+* TODO: Tierney volunteered to open issue in Admin to propose moving repo into the nodejs org
+* TODO: issue in TSC for awareness - https://github.com/nodejs/TSC/issues/443
+
+### Node.js Strategic Initiatives beyond technical matters [186] (https://github.com/nodejs/community-committee/issues/186)
```
Comment (pup) [inline meetings/2017-12-14.md]: this link doesn't render correctly

**Comment by bnb** [comment]
Going to merge this in - addressed @pup's comments 👍

**Pull request merged**